serving/docker/lmi.Dockerfile

```diff
@@ -33,7 +33,7 @@ ARG auto_gptq_version=0.7.1
 ARG datasets_version=2.19.1
 ARG autoawq_version=0.2.5
 # LMI-Dist Deps
-ARG vllm_wheel="https://publish.djl.ai/vllm/cu124-pt230/vllm-0.4.2%2Bcu124-cp310-cp310-linux_x86_64.whl"
+ARG vllm_wheel="https://github.com/vllm-project/vllm/releases/download/v0.5.1/vllm-0.5.1+cu118-cp310-cp310-manylinux1_x86_64.whl"
```

Comment:
there is a problem here, we should not use the CU118 wheel

Reply:
```
https://github.com/vllm-project/vllm/releases/download/v0.5.1/vllm-0.5.1-cp310-cp310-manylinux1_x86_64.whl
```

Reply:
Done.